deepmark_chainer/net/inception_v3.py

```diff
@@ -235,4 +236,21 @@ def classifier(x, train):
         y = classifier(x, train)
         self.y = y
         self.y_aux = y_aux
-        return y
+        return y, y_aux
+
+
+class Classifier(link.Chain):
```

Comment:
This uses the same name as the Classifier in Chainer. Consider renaming to something more specific for clarity.